reconcile/utils/sentry_client.py

```diff
@@ -27,10 +28,35 @@ def _do_sentry_api_call_(self, method, path, slugs, payload=None):
         response.raise_for_status()
 
         try:
-            return response.json()
+            all_results = response.json()
         except json.decoder.JSONDecodeError:
             return
 
+        # there may be more pages if the response contains a link header
+        # link is a string of comma separated items
+        # with the following structure:
+        # <URL>; rel="previous/next"; results="false/true"; cursor="value"
+        item_format = '<{}>; rel="{}"; results="{}"; cursor="{}"'
+        while True:
+            link = response.headers.get('link')
+            if not link:
+                break
+            # 2nd item is the next page
+            next_item = link.split(', ')[1]
+            # copied with love from
+            # https://stackoverflow.com/questions/10663093/
+            # use-python-format-string-in-reverse-for-parsing
+            _, rel, results, cursor = parse.parse(item_format, next_item)
+            if rel != 'next' or results != 'true':
+                break
+            response = \
+                call(f"{url}?&cursor={cursor}", headers=headers, json=payload)
```

Comment:
Curious: in what situations does 'call' fail? Is it worth wrapping it in a try/catch block?

Reply:
the way to test the result of call (`requests.get` for example) is with `response.raise_for_status()` (which will raise an exception we want to see if something happened).